src/Core/AdminConsole/OrganizationFeatures/Policies/PolicyRequirements/ResetPasswordPolicyRequirement.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Bit.Core.AdminConsole.Enums;
+using Bit.Core.AdminConsole.Models.Data.Organizations.Policies;
+
+namespace Bit.Core.AdminConsole.OrganizationFeatures.Policies.PolicyRequirements;
+
+/// <summary>
+/// Policy requirements for the Account recovery administration policy.
+/// </summary>
+public class ResetPasswordPolicyRequirement : IPolicyRequirement
+{
+    /// <summary>
+    /// Indicates that new members will automatically be enrolled in account recovery administration when joining an organization.
+    /// </summary>
+    public bool AutoEnrollEnabled { get; init; }
+}
+
+public class ResetPasswordPolicyRequirementFactory : BasePolicyRequirementFactory<ResetPasswordPolicyRequirement>
```

Comment:
Because we use all direct repository calls today, no-one is exempt due to their role or provider status.

To maintain current behaviour, let's override the `ExemptRoles => []` and `ExemptProviders => false`.